pagopa/pn-frontend
# Feat/pn 12056

## Short description
Created two components fro email and sms contact

## List of changes proposed in this pull request
- Created EmailContactItem and SmsContactItem
- Split test files

## How to test
Login with pf/pg -> go to Recapiti page -> add/edit/remove sms and email contact and check that everything is ok
